config/manager/controller-manager.yaml

```diff
@@ -83,9 +83,14 @@ spec:
         volumeMounts: 
           - name: config-volume
             mountPath: /etc/numaplane
+          - name: numarollout-volume
+            mountPath: /etc/numarollout
       serviceAccountName: numaplane-sa
       terminationGracePeriodSeconds: 10
       volumes:
         - name: config-volume
           configMap:
             name: numaplane-controller-config
+        - name: numarollout-volume
+          configMap:
+            name: numarollout-config
```

Comment:
yeah, if we volume mount the file, you're right that it probably makes the most sense to have just a single file with one key for each version.

(An alternative we could potentially weigh in the future, if for some reason it's better to have multiple files, is to make a call to K8S API Server to Get all of the ConfigMaps with a particular label or something. I'm just thinking "what's the easiest way from the `numa-manifest-generator` side to create everything?" But maybe one file still works there as well...hmm)